__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const request = require("supertest");
+const app = require("../app");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+    test("GET:200 sends an array of topics to the client", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => { console.log(body.topics)
+            const { topics } = body;
+            topics.forEach((topic) => {
```

Comment:
before running forEach, we should be running an assertion to check the length of the topics that are returned from the database. The reason for this is that if an empty array came back, the forEach would never execute it's callback, and therefore we would never run the test assertion inside that callback, and we would end up with a false positive for this test.